plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java

```diff
@@ -64,6 +87,63 @@ public static Library getLibrary(String id) {
     return libraries.get(id);
   }
 
+  private final static Supplier<List<Library>> clientApis = Suppliers.memoize(new Supplier<List<Library>>() {
+    public List<Library> get() {
+      Bundle bundle = FrameworkUtil.getBundle(CloudSdk.class);
+      URL url = bundle.getResource("/com/google/cloud/tools/libraries/libraries.json");
+      
+      try (InputStream in = url.openStream()) {
+        JsonReaderFactory factory = Json.createReaderFactory(null);
+        JsonReader reader = factory.createReader(in); 
```

Comment:
Turns out we can by-pass the factory and call the static method `Json.createReader(in)` directly.

Reply:
neat. done.